jenkins_workspace_cleanup.sh

```diff
@@ -0,0 +1,29 @@
+# /*
+# (C) Copyright ${year} Nuxeo (http://nuxeo.com/) and others.
+# *
+# * Licensed under the Apache License, Version 2.0 (the "License");
+# * you may not use this file except in compliance with the License.
+# * You may obtain a copy of the License at
+# *
+# *     http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# *
+# * Contributors:
+# *     ...
+# */
+
+###### Cron file for jenkins qa* ######
+
+#!/bin/bash
+
+docker image prune -f > /dev/null
+docker volume prune -f > /dev/null
+docker ps -a | grep Exited | grep "hours ago" | awk '{print $1}' | xargs --no-run-if-empty docker rm -v > /dev/null 2>&1
```

Comment:
You should merge the two `grep`s, and `awk` can match like `grep` as well:

    docker ps -a | awk '/Exited.*hours ago/ {print $1}' | ...

Note that it can say `weeks ago` too, and probably `days ago`. So I'd use `Exited.*(hours|days|weeks) ago`